.github/workflows/benchmark/terraform/variables.tf

```diff
@@ -10,13 +10,13 @@ variable "secret_key" {
 
 variable "region" {
   type    = string
-  default = "ap-northeast-1"
+  default = "us-east-2"
```

Comment:
It may timeout for github-hosted runner to connect to ap-northeast-1 ec2, so I changed to us-east-2